cmd/web-handlers.go

```diff
@@ -129,7 +129,13 @@ func (web *webAPIHandlers) StorageInfo(r *http.Request, args *WebGenericArgs, re
 	if authErr != nil {
 		return toJSONError(ctx, authErr)
 	}
-	reply.StorageInfo, _ = objectAPI.StorageInfo(ctx, false)
+	// StorageInfo calculation in distributed setup
+	dataUsageInfo, err := loadDataUsageFromBackend(ctx, objectAPI)
+	if err != nil {
+		return toJSONError(ctx, errServerNotInitialized)
```

Comment:
Why are we returning a fixed error?

Reply:
We shouldn't do this @BigUstad - just populate an empty value which is good enough